### PR TITLE
EOS-23367: Utils-specific-supportbundle: Added kafka log files into support bundle tar

### PR DESCRIPTION
**Pull request by suryakumar1024**
Signed-off-by: suryakumar.kumaravelan <suryakumar.kumaravelan@seagate.com>

# Problem Statement
- Added log files from /opt/kafka/logs dir

# Design
-  For Bug describe the fix here. 
-  For Feature, Post the link to the solution page on the confluence CORTX Foundation Library 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/support/utils_support_bundle.py [hunk 1]
```diff
@@ -111,6 +105,11 @@ def __collect_kafka_logs():
                 if value and os.path.exists(value):
                     shutil.copytree(value,
                         os.path.join(UtilsSupportBundle._tmp_src, key))
+        # Copy all required log files from /opt/kafka/logs directory
+        if os.path.exists('/opt/kafka/logs/'):
```
Comment (ujjwalpl) [inline py-utils/src/support/utils_support_bundle.py:109]: Why is Kafka logs going to /opt/ It should go to /var/log only. Can we fix that ?

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: py-utils/src/support/utils_support_bundle.py [hunk 2]
```diff
@@ -111,6 +105,11 @@ def __collect_kafka_logs():
                 if value and os.path.exists(value):
                     shutil.copytree(value,
                         os.path.join(UtilsSupportBundle._tmp_src, key))
+        # Copy all required log files from /var/log/kafka directory
+        if os.path.exists('/var/log/kafka/'):
```
Comment (ujjwalpl) [inline py-utils/src/support/utils_support_bundle.py:109]: Please do not hard code. Log path may change to /share/var/log. So we should be prepared to get this as config input.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

**Review by ujjwalpl — APPROVED** [review]

File: py-utils/src/support/utils_support_bundle.py [hunk 3]
```diff
@@ -111,6 +105,11 @@ def __collect_kafka_logs():
...
+            ignore =shutil.ignore_patterns('*.log.*-*-*-*')
```
Comment (selvakumaar5496) [inline py-utils/src/support/utils_support_bundle.py:110]: provide a space after =

File: py-utils/src/support/utils_support_bundle.py [hunk 4]
```diff
@@ -111,6 +105,11 @@ def __collect_kafka_logs():
...
+            shutil.copytree('/var/log/kafka/', os.path.join(
+                UtilsSupportBundle._tmp_src, 'kafka_log') , ignore=ignore)
```
Comment (selvakumaar5496) [inline py-utils/src/support/utils_support_bundle.py:112]: remove space before comma

File: py-utils/src/support/utils_support_bundle.py (hunk 3 above)
Comment (suryakumar1024) [reply py-utils/src/support/utils_support_bundle.py]: updated in recent patch

File: py-utils/src/support/utils_support_bundle.py (hunk 4 above)
Comment (suryakumar1024) [reply py-utils/src/support/utils_support_bundle.py]: updated in recent patch

**Review by selvakumaar5496 — APPROVED** [review]

**Review by rohit-k-dwivedi — APPROVED** [review]

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**